examples/nono-sandbox/workload.py

```
@@ -0,0 +1,217 @@
+    "--quiet",
+    "--output=raw",
+    "query",
+    "--since=15m",
+    "--limit=10",
+    INCIDENT_QUERY,
+)
+
+
+def run_logcli(*args):
+    """Invoke by command name so nono's tool-sandbox shim mediates the call."""
+    return subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)